src/core/converters/type_to_yaml.py

```
@@ -881,115 +880,102 @@ def types_to_yaml_simple(
-        # 型オブジェクトの場合(従来の処理)
-        # クラスのdocstringを取得
-        docstring = _get_docstring(typ)
-        if docstring:
-            # 複数行のdocstringはヒアドキュメント形式(| 形式)で出力
-            if "\n" in docstring:
-                type_data["description"] = LiteralScalarString(docstring)
-            else:
-                type_data["description"] = docstring
+        else:
+            # 型オブジェクトの場合(従来の処理)
+            # この時点で typ は type[Any] であることが保証される（AST型チェックで除外済み）
```

Ruff (0.14.0)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)